buildAll.bat

```diff
@@ -11,8 +11,25 @@ call tools\build.bat vioscsi\vioscsi.vcxproj "Win8_SDV Win10_SDV" %*
 if errorlevel 1 goto :fail
 call tools\build.bat viostor\viostor.vcxproj "Win8_SDV Win10_SDV" %*
 if errorlevel 1 goto :fail
+if "%VIRTIO_WIN_SDV_2022%"=="" goto :nosdv2022
```

Comment:
I think buildAll.bat should always run SDV tests. 
@YanVugenfirer what do you think?

Reply:
> I think buildAll.bat should always run SDV tests.
> @YanVugenfirer what do you think?

I am not sure yet.
Depending on if we want to break the build into two steps or not.
For now, I would prefer to keep it as it is in the patch.

Reply:
VIRTIO_WIN_SDV_2022 variable looks very specifically. We also will run all SDV for Windows 11. Maybe better to use VIRTIO_WIN_ALL_SDV or something like this?

Reply:
The first idea was to enable SDV only for drivers that really need to have valid DVL to pass Win2022 Static Logo Test. Because the static analysis takes quite a long time, and here, at Virtuozzo, we found that DVL for the Display class drivers, for example, is still not required. But if all decide that now we will always run SDV for each driver, then it is reasonable to use VIRTIO_WIN_ALL_SDV.